tools/flash_shelly.py

```diff
@@ -109,6 +109,10 @@ def get_info(host):
       info['model'] = shelly_model(info['app'])
     info['fw_type'] = 'homekit'
     info['fw_type_str'] = 'HomeKit'
+    if info['model']  == 'ShellyRGBW2':
```

Comment:
this condition has the same code in both branches

Reply:
it a placeholder for when we do HomeKit RGBW2, i dont want to forget, i can comment out if you want ?

Reply:
tweaked https://github.com/mongoose-os-apps/shelly-homekit/pull/292/commits/624c66c05f4d9bfb5aa9837cf450ce55290416f0